e2e/pages/organizations/developers/developers.readonly.spec.js

```diff
@@ -76,3 +84,20 @@ describe('the Developers page', () => {
         });
     });
 });
+
+describe('When user is on the Developer page for a Developer that doesnt exist', () => {
+    beforeEach(async () => {
+        browser.setWindowSize(1600, 1024);
+        browser.setWindowRect(0, 0, 1600, 1024);
+        contact = new ContactComponent();
+        page = new DevelopersPage();
+        hooks = new Hooks();
+        const dummyDeveloperId = '0000';
+        await hooks.open('#/organizations/developers/' + dummyDeveloperId);
+    });
+
+    it('it should go to Home page', () => {
```

Comment:
```suggestion
    it('should go to Home page', () => {
```

Don't need the "double it"